src/api/content-strings/fields/insurance/your-buyer/index.ts

```diff
@@ -106,7 +106,7 @@ export const YOUR_BUYER_FIELDS = {
     },
   },
   [CURRENCY_CODE]: {
-    LEGEND: 'What currency are the outstanding or overdue payments in?',
+    LEGEND: 'Select the currency the outstanding or overdue payments are in?',
```

Comment:
Can we check the design for this, doesn't seem it should have a question mark? ("Select the..." as opposed to e.g "What is the...")

Reply:
the design has the question mark and select

Reply:
Feels strange to me - pinged design, not a blocker.

Reply:
Design confirmed, need to update this